pkg/cluster/hash_test.go

```diff
@@ -185,6 +185,23 @@ func TestHashScaleSetStability(t *testing.T) {
 				expectedHash: "959b10157d2864150339914d0e5ddee5dc7d7e1e6d2318f0ef9e5325cc6b09df",
 			},
 		},
+		"v13.0": {
```

Comment:
Confused by this should this not be adding a 13.1 here looking at the file there are hashes for eg 12.2. Also do you need to add the 13.1 case in TestHashScaleSetStability earlier in this file

Reply:
This test ensures that we don't accidentally modify the contents of 13.0 now that it's been released; it's okay for us to modify the contents of 13.1 until we officially release and tag it. (I lifted the hashes from master, where 13.0 was already frozen.)

Reply:
The unit tests require us to have hashes for every version release except for the current `pluginVersion` in the pluginconfig: 

https://github.com/openshift/openshift-azure/blob/ffed473e54f737db539a0bfa9e1a4939d542f45f/pkg/cluster/hash_test.go#L226-L230